src/main/java/games/strategy/engine/framework/GameDataFileUtils.java

```diff
@@ -9,21 +9,22 @@
 
 import com.google.common.annotations.VisibleForTesting;
 
-import games.strategy.engine.ClientContext;
+import games.strategy.triplea.settings.ClientSetting;
 
 /**
  * A collection of utilities for working with game data files.
  */
 public final class GameDataFileUtils {
   private static final IOCase DEFAULT_IO_CASE = IOCase.SYSTEM;
 
-  private static final SaveGameFormat DEFAULT_SAVE_GAME_FORMAT =
-      ClientContext.gameEnginePropertyReader().useNewSaveGameFormat()
-          ? SaveGameFormat.NEW
-          : SaveGameFormat.CURRENT;
-
   private GameDataFileUtils() {}
 
+  private static SaveGameFormat getDefaultSaveGameFormat() {
+    return ClientSetting.TEST_USE_NEW_SAVE_GAME_FORMAT.booleanValue()
+        ? SaveGameFormat.NEW
```

Comment:
I struggled with naming the "current" and "new" save game formats in the PR where the feature toggle was first introduced.  As your suggestions demonstrate, the names became somewhat verbose.  I ultimately gave up when I convinced myself that all of this code referring to different save game formats is transient and will be removed in the next major release when we fully switch to the new format.

However, I'll give it another shot and see if I can come up with more meaningful names to replace "new" and "current" in this class and `GameDataManager`.  I'll do that in a follow-up PR.